sql_generators/experiment_monitoring/templates/experiment_crash_rates_live/view.sql

```diff
@@ -0,0 +1,48 @@
+-- Generated via ./bqetl generate experiment_monitoring
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.experiment_crash_rates_live`
+AS
+WITH crash_events AS (
+{% for app_dataset in applications %}
+  SELECT
+    experiment,
+    branch,
+    window_start,
+    window_end,
+    crash_process_type,
+    crash_count
+  FROM
+    `moz-fx-data-shared-prod.{{ app_dataset }}_derived.experiment_crash_events_live_v1`
+  WHERE
+    window_start > TIMESTAMP(DATE_SUB(CURRENT_DATE(), INTERVAL 2 DAY))
+  UNION ALL
+{% endfor %}
+SELECT
+  experiment,
+  branch,
+  window_start,
+  window_end,
+  crash_process_type,
+  crash_count
+FROM
+  `moz-fx-data-shared-prod.telemetry_derived.experiment_crash_aggregates_v1`
+WHERE
+  window_start <= TIMESTAMP(DATE_SUB(CURRENT_DATE(), INTERVAL 2 DAY))
+)
+SELECT
+  crash_events.experiment AS experiment,
+  crash_events.branch AS branch,
+  window_start,
+  window_end,
+  crash_process_type,
+  crash_count,
+  crash_count / enrollments.value AS estimated_crash_rate
```

Comment:
Crash counts and crash count rates are for 5-minute long time windows. The number of crashes seems to be quite low (approx. <20 for the ones I looked at), so maybe just showing the total number of crashes is good enough for experiment owners to see that something is off. But keeping the crash rate metric for now in case it turns out to be useful